app/controllers/users_controller.rb

```diff
@@ -17,7 +20,8 @@ def show
     @item = Item.new
   end
 
-  def new
+  def index
```

Comment:
@fuzzyalej Haha I leaved that there by mistake, I was trying to show a list of users when testing the `:user_id` doubt I had yesterday. There we go: https://github.com/ivorpad/blocitoff/commit/d221f13ba06319eb41a334421d6a5002f852e4c3